x-pack/plugins/lists/common/test_utils.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as t from 'io-ts';
+import { fold } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+
+import { formatErrors } from './format_errors';
+
+interface Message<T> {
+  errors: t.Errors;
+  schema: T | {};
+}
+
+const onLeft = <T>(errors: t.Errors): Message<T> => {
+  return { errors, schema: {} };
+};
+
+const onRight = <T>(schema: T): Message<T> => {
+  return {
+    errors: [],
+    schema,
+  };
+};
+
+export const foldLeftRight = fold(onLeft, onRight);
+
+/**
+ * Convenience utility to keep the error message handling within tests to be
+ * very concise.
+ * @param validation The validation to get the errors from
+ */
+export const getPaths = <A>(validation: t.Validation<A>): string[] => {
```

Comment:
These are redefined from `security_solution`. Should we move these out to a common location?

Reply:
Yeah, I would like to do that. I'm not sure the best way of sharing code between plugins like these and I know there's been some work to decouple lists from `security_solution`, so will have to check with @oatkiller on this as he's been involved in that a little.